src/coins.h

```diff
@@ -184,13 +177,13 @@ struct CCoinsCacheEntry
     inline bool IsDirty() const noexcept { return m_flags & DIRTY; }
     inline bool IsFresh() const noexcept { return m_flags & FRESH; }
 
-    //! Only call Next when this entry is DIRTY, FRESH, or both
+    //! Only call Next when this entry is DIRTY
     inline CoinsCachePair* Next() const noexcept {
         Assume(m_flags);
```

Comment:
```suggestion
        Assume(IsDirty());
```

nit: it's a cheap call, why not make it an assert instead?

Reply:
Done.